tests/acceptance/test_organization_integration.py

```diff
@@ -0,0 +1,150 @@
+)
+
+
+class OrganizationIntegrationAcceptanceTestCase(AcceptanceTestCase):
+    def setUp(self):
+        super(OrganizationIntegrationAcceptanceTestCase, self).setUp()
+        self.user = self.create_user('foo@example.com')
+        self.org = self.create_organization(
+            name='Rowdy Tiger',
+            owner=None,
+        )
+        self.team = self.create_team(organization=self.org, name='Mariachi Band')
+        self.project = self.create_project(
+            organization=self.org,
+            teams=[self.team],
+            name='Bengal',
+        )
+        self.create_member(
+            user=self.user,
+            organization=self.org,
+            role='owner',
+            teams=[self.team],
+        )
+        self.model = Integration.objects.create(
+            provider='example',
+            external_id='example',
+            name='Test Integration',
+            metadata={
+                'domain_name': 'example-test.com',
+            },
+        )
+        self.org_integration = self.model.add_organization(self.org, self.user)
+        self.login_as(self.user)
+
+        self.integration_settings_path = 'sentry-api-0-organization-integrations'
+
+    def create_user_with_role(self, role, organization=None, teams=None):
+        """
+        Owner, manager, admin
+        """
+        user = self.create_user()
+        self.create_member(
+            organization=organization if organization else self.organization,
+            user=user,
+            role=role,
+            teams=teams if teams else [],
+        )
+        return user
+
+    def load_page(self, url):
+        self.browser.get(url)
+        self.browser.wait_until_not('.loading-indicator')
+
+    def assert_button_element(self, button, label, disabled=False, icon=None):
+        disabled = 'true' if disabled else 'false'
+        assert button.get_disabled() == disabled
+        assert button.get_label() == label
+        if icon:
+            assert button.get_icon_href() == icon
+
+
+class OrganizationIntegrationSettingsTest(OrganizationIntegrationAcceptanceTestCase):
+    """
+    As a user(type?), I can setup, configure, and remove an integration.
+    """
+    # TODO(lb): Tests to be written
+    # test_setup_new_integration_with_repository
+    # test_setup_new_integration_with_issue_sync
+    # test_remove_existing_integration_installation
+    # test_update_legacy_integration
+    # test_user_permissions_for_integration_settings
+    # test_add_multiple_integrations_to_one_provider
+
+    def setUp(self):
+        super(OrganizationIntegrationSettingsTest, self).setUp()
+        self.org_integration_settings_path = u'/settings/{}/integrations/'.format(
+            self.organization.slug)
+
+        self.provider = mock.Mock()
+        self.provider.key = 'example'
+        self.provider.name = 'Example Installation'
+
+    def assert_can_create_new_installation(self):
```

Comment:
@markstory  Didn't add any of the previous assertions yet. Wanted to know what you thought about this instead, before adding more. Let me know what you think

Reply:
I think the contents of the method are great. If you're not going to use it multiple times the name could be `test_create_new_installation` though.